web-client/src/app/views/delete-user/delete-user.page.html

```diff
@@ -7,24 +7,47 @@
   </ion-toolbar>
 </ion-header>
 
-<ion-content>
+<ion-content *ngIf="hideXrpBalance; else Default">
   <div class="ion-text-center p-2">
     <p>
-      You will need to enter a wallet address to send your remaining XRP balance
-      to.
+      You will need to send your remaining {{tokenSign}} balance before you can
```

Comment:
will make sense to show this message when the digital asset amount is not zero

Reply:
Thanks! In the latest commit, I've added another message to show when the issued currency balance is zero.